src/api/services/groups.js

```diff
@@ -310,10 +311,40 @@ module.exports.getGroupPrintable = async (options) => {
 };
 
 
-let toPDF = async (title, participants) => {
-  return new Promise((resolve, reject) => {
-    var pdf = require('html-pdf');
+async function convertHTMLtoPDF(htmlContent) {
```

Comment:
Remove ASAP, just generate a simple plain html page with a specific CSS that it is also used for @media print. Users can use browsers' native feature to save as PDF to generate a PDF if required,  or just to printout the tokens.

Reply:
fixed: Done in frontend